src/components/Download.tsx

```diff
@@ -67,7 +67,7 @@ export default function Download({ eventIds }: { eventIds: string[] }): JSX.Elem
   };
 
   return (
-    <div className="p-4">
+    <div>
```

Comment:
Just as an FYI, I think the padding was present to prevent the overlap of content.

![image](https://github.com/user-attachments/assets/ece7c583-c43e-49a1-986f-8df82c1b24a3)

Reply:
Do you have a suggestion on how to fix this? I guess I could try and move the tabs but the table would look funny if it had enough padding to prevent the overlap.

Reply:
The padding was my solution. I can't think of any other way to prevent overlap unless we move the default trigger to a button in the drawer and remove the external button.